minitwit/src/main/java/WebApplication.java

```diff
@@ -43,11 +46,27 @@ public static void main(String[] args) {
         get("/hello", (req, res) -> "Hello");
         get(URLS.PUBLIC_TIMELINE, WebApplication.servePublicTimelinePage);
         get(URLS.REGISTER, WebApplication.serveRegisterPage);
+        get(URLS.LOGIN, WebApplication.serveLoginPage);
+        post(URLS.LOGOUT, handleLogoutRequest);
```

Comment:
Any reason this is not `WebApplication.handleLogoutRequest`

Reply:
No, just an oversight on my part.